code/modules/clothing/glasses/glasses.dm

```diff
@@ -4,7 +4,7 @@
 	desc = "Optical Meson Scanner with prescription lenses."
 	prescription = 1
 	eyeprot = -1
-	species_fit = list("Vox")
+	species_fit = list("Vox","Skeletal Vox")
```

Comment:
@ComicIronic Pretty sure he'd have to refactor the procs to handle species_fit though since I'm guessing it just compares the species datum's string to this